subsystem/kubeletconfig_resource_test.go

```diff
@@ -82,6 +82,20 @@ var _ = Describe("Cluster KubeletConfig", func() {
 
 		It("fails if OCM backend fails to create the object", func() {
```

Comment:
@marcolan018 We should extend this test to cover HCP clusters as well, particularly ensuring we allow the user to create multiple kubeletconfigs.

Reply:
new subsystem tests added for kubelet configs